common/test/acceptance/tests/studio/test_import_export.py

```diff
@@ -12,6 +12,25 @@
 
 
 # pylint: disable=no-member
+class TemplateTestMixin(object):
```

Comment:
@Kelketek - I'd like to see what other people have to say about this - maybe @chrisndodge and/or @mattdrayer would chime in.

Personally, I'd prefer a single test added to each test class
e.g. in ```TestCourseExport```
add
```
def test_header(self):
    self.assertEqual(self.template_page.header_text, 'Course Export');
```

Then it's obvious that you are testing that the page has the header 'Course Export'. And so on for the other test classes...

No matter how much help one gets from the IDE - the fact that it is pretty plainly stated in the test is much more compelling to me - and this would result in 8 new lines of code instead of 11!